.eslintrc.js

```diff
@@ -64,8 +69,18 @@ module.exports = {
         'no-unused-expressions': 'off',
         'mocha/no-exclusive-tests': 'error',
         'mocha/no-mocha-arrows': 'error',
+        'sort-keys': 'off',
```

Comment:
Why is `sort-keys` off for testcafé?

Reply:
Good question...I'm not sure I fully remember. I think it might have been because there were _a lot_ of warnings to address so I left it off this PR...